### PR TITLE
Monitoring: Alerts and filter workload metrics

### PR DESCRIPTION
**Pull request by Preeticp**
This PR fixes https://issues.redhat.com/browse/RHDEVDOCS-2075 and https://issues.redhat.com/browse/RHDEVDOCS-2083
It is slated for 4.6
It has been reviewed and approved by the SME.

### CODE REVIEW

**Comment by Preeticp** [comment]
@debsmita1 @gajanan-more this PR is ready for SME and QE review. Please let me know if we need to modify for add anything more.
I just have one question wrt the permissions for the Notifications toggle (I have commented out the question in the section). If I can get an answer to that, I will update that too. Thank you!

File: applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc [hunk 1]
```diff
@@ -15,3 +15,6 @@ The *Monitoring* view in the *Developer* perspective provides options to monitor
 include::modules/odc-monitoring-your-project-metrics.adoc[leveloffset=+1]
 
 include::modules/odc-monitoring-your-application-metrics.adoc[leveloffset=+1]
+
+== Additional Resources
+* xref:../monitoring/cluster_monitoring/about-cluster-monitoring.adoc[Monitoring cluster metrics]
```
Comment (ahardin-rh) [inline applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc:20]: * xref:../monitoring/cluster_monitoring/about-cluster-monitoring..adoc#about-cluster-monitoring[[Monitoring cluster metrics]

File: modules/odc-monitoring-your-application-metrics.adoc [hunk 2]
```diff
@@ -5,15 +5,17 @@
...
-After you create applications in your project and deploy them, you can use the *Topology* view in the *Developer* perspective to see the metrics for your application.
+After you create applications in your project and deploy them, you can use the *Topology* view in the *Developer* perspective to see the alerts and metrics for your application. Critical and warning alerts for your application are indicated on the workload node in the *Topology* view.
 
 .Procedure
+To see the alerts for your workload:
 
-. In the *Topology* view, click the application to see the application details in the right panel.
-. Click the *Monitoring* tab to see the warning events for the application; graphs for CPU, memory, and bandwidth usage; and all the events for the application.
+. In the *Topology* view, click the workload to see the workload details in the right panel.
+. Click the *Monitoring* tab to see the critical and warning alerts fired for the application; graphs for metrics, such as CPU, memory, and bandwidth usage; and all the events for the application.
```
Comment (ahardin-rh) [inline modules/odc-monitoring-your-application-metrics.adoc:14]: s/alerts fired for/alerts for

File: modules/odc-monitoring-your-project-metrics.adoc [hunk 3]
```diff
@@ -34,6 +35,20 @@ image::odc_project_metrics.png[]
...
++
+.Monitoring alerts
+image::odc_project_alerts.png[]
++
+** Use the *Filter* list to filter the alerts by their state and severity.
+
+** Click on an alert to go to the details page for that alert. In the *Alerts Details* page, you can click the *View Metrics* button to see the metrics for the alert.
+
+** Use the *Notifications* toggle adjoining an alert to silence the alert, and then select the duration for which the alert will be silenced from the *Silence for* list.
+Note that you need to have the permissions to edit alerts to see the *Notifications* toggle.
```
Comment (ahardin-rh) [inline modules/odc-monitoring-your-project-metrics.adoc:48]: s/Note that you need to have the/You must have the

**Comment by ahardin-rh** [comment]
@Preeticp Looks good! Just a few minor comments from me. Once those are addressed and the commits are squashed, this should be ready to go. Thanks!

File: modules/odc-monitoring-your-application-metrics.adoc (hunk 2 above)
Comment (Preeticp) [reply modules/odc-monitoring-your-application-metrics.adoc]: Firing is a state of an alert. I have added a note for the same, and also listed the states that are not displayed to make it clear.

**Comment by Preeticp** [comment]
@ahardin-rh thank you. I have fixed all the review comments and squashed the comments. PTAL and merge if it looks good.

**Comment by Preeticp** [comment]
@vikram-raj I have fixed the comments, PTAL and add a LGTM if you are good with the changes. TY!!

**Review by vikram-raj — COMMENTED** [review]
Thanks @Preeticp.

/lgtm

**Comment by Preeticp** [comment]
Hi @ahardin-rh I received three minor comments from another SME, who was on PTO earlier. I have fixed those. Request you to PTAL and merge if it looks good.

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**